src/patternfly/components/Table/table-grid.scss

```diff
@@ -92,7 +92,6 @@
 
   // Table check
   --pf-c-table__check--responsive--MarginLeft: var(--pf-global--spacer--sm);
-  --pf-c-table__check--responsive--MarginTop: #{pf-size-prem(6px)};
```

Comment:
We have to leave variables in place and not rename them, otherwise the change is considered breaking, since a user could have overridden this variable in their local app, and if we remove it, their override no longer applies. Instead of removing it, you can apply a value of "0". You'll also need to add back the `margin-top` declaration that references it.

Reply:
Ok. I've added it back in.